.travis.yml

```diff
@@ -132,10 +135,11 @@ install:
     # install was successful by trying to import the toolkit (sometimes, the
     # install appears to be successful but shared libraries cannot be loaded at
     # runtime, so an actual import is a better check).
-    python -mpip install --upgrade cairocffi>=0.8 pgi>=0.0.11.2 &&
-      python -c 'import pgi as gi; gi.require_version("Gtk", "3.0"); from pgi.repository import Gtk' &&
-      echo 'pgi is available' ||
-      echo 'pgi is not available'
+    python -mpip install --upgrade pycairo cairocffi>=0.8
+    python -mpip install --upgrade PyGObject==3.30.4 &&
```

Comment:
Is there a reason PyGObject is pinned to this specific version?

Reply:
Just the latest stable release to keep the CI environment stable and in case PyGObject drops Ubuntu 16.04 support, even though there are no plans to do so right now.

I can remove it if you want.

Reply:
Not quite sure on our version policy for CI. Keeping it stable and always testing the mos recent one both have their benefits. Nevertheless, I would go with the most recent.

@tacaswell Do you have an opinion on version pinning here?

Reply:
I've removed the pinning now.

Reply:
a belated :+1: to removing the pinning.  Would rather deal with broken CI than finding out after things are released.